kafka-streams/src/main/kotlin/com/example/kafkastreams/model/TimingsSummary.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.kafkastreams.model
+
+import java.time.Instant
+
+data class TimingsSummary (
```

Comment:
Any reason these are nullable?

Reply:
It's because when I call `val min = allTimings.minOrNull()` in the MessageController. Originally I tried `min()` but it told me it's deprecated and I should use `minOrNull()`. I tried to look around it but couldn't find anything obvious